### ISSUE TITLE
Search Panel is missing progress bar and showing badges during search

### ISSUE THREAD

**Issue by sbatten** [microsoft/vscode#85957]
Reproduces in stable 1.40.x

Steps to Reproduce:

1. Open VS Code
2. Toggle search position **at least once** until search is present in panel.
3. Try a search

:bug: => see image

![image](https://user-images.githubusercontent.com/6561887/69989724-2053da80-14f9-11ea-96c2-729db1dd9bd5.png)

Note that if you start VS Code with search in panel and don't toggle the position, the issue will not reproduce.

**Comment by isidorn** [comment]
Since it reproduces in stable and happens only once you switch it does not sound critical enough.
Also it might be fixed with your refactorings and changes. Thus I am not so keen to jupm on it immediatly.
The prograss bar was probably registered in the viewlet area, and thus when we search was moved we did not . Good code pointer https://github.com/Microsoft/vscode/blob/56c6acd2fe1a2b7f53d4fd0cf0dfcc27837eb585/src/vs/workbench/services/progress/browser/progressService.ts#L51
So probably the `viewletService` `getProgressIndicator` still returns an `IProgressIndicator` even if the search was moved.
